exporter/awsemfexporter/emf_exporter.go

```diff
@@ -82,7 +81,7 @@ func newEmfPusher(
 	svcStructuredLog := cwlogs.NewClient(logger, awsConfig, params.BuildInfo, expConfig.LogGroupName, session)
 	collectorIdentifier, _ := uuid.NewRandom()
 
-	expConfig.Validate()
+	_ = expConfig.Validate()
```

Comment:
Same here, this duplicates what is already called on the service
```suggestion
```

Reply:
Ok, I misunderstood